packages/flutter_tools/lib/src/run_cold.dart

```diff
@@ -185,10 +185,8 @@ class ColdRunner extends ResidentRunner {
   @override
   void printHelp({ @required bool details }) {
     globals.printStatus('Flutter run key commands.');
-    if (supportsServiceProtocol) {
-      if (details) {
-        printHelpDetails();
-      }
+    if (details) {
```

Comment:
Does this change the help text for release mode?

Reply:
Yes, the screenshot help was missing despite the fact that it should be supported. I added a test case for this.

Reply:
Sorry, I'm confused. It's supported in profile but not in release, right? Is `supportsServiceProtocol` what we were using to distinguish those cases?

Reply:
its supported in profile and release, because it involves talking to a device daemon (ios-deploy/adb) and not the service protocol.

Reply:
actually idevicescreenshot, not ios-deploy